source/radmon_flag.rst

```diff
@@ -0,0 +1,120 @@
+.. _radmon_flag:
+
+Anomalous Radiation Monitor Flag Assertion
+===========================================
+
+(This anomaly page is a work in progress.)
+
+What is it?
+-----------
+
+The ACIS Radiation Monitor Flag was asserted anomalously, presumably due to a spurious command.  This suspends any active science runs and powers off the video boards.
+
+When did it happen before?
+--------------------------
+
+This anomaly has not previously occurred.
+
+Will it happen again?
+---------------------
+
+There have been no spurious hardware serial digital commands in ACIS to date.  (Peter should verify this statement!)
+
+How is this anomaly diagnosed?
+------------------------------
+
+If the radmon flag is asserted during real-time contact with ACIS,
+
+* In software housekeeping,
+
+  * SWSTAT_SCI_INHIBIT_ON = 74
+  * SWSTAT_DEACCD_POWEROFF = 81
+
+* If a science run was in progress, terminationCode = SMTERM_RADMON in the scienceReport
+
+While science is inhibited:
+
+* Bilevels are (science idle or science active, PGF & JF need to check on EU)
+* a dump of sysConfig would show all video boards down and FEPs in the correct configuration from the command load
+* DEA current consistent with video boards off (Need to add value)
+* No event data, even if the command load implies there should be
+* If a start science command is issued, command echo result = 13 (CMDRESULT_INHIBITED)
+* If a stop science command is issued, command echo result = 13 (CMDRESULT_INHIBITED)
+
+What is the response?
+---------------------
+
+(will this show up in a standard SOH by OC/CC?  Don't think so but
+need to check)
+We need to:
+
+* Send an email to the ACIS team at the official anomaly email address, ``acis-anomaly -at- googlegroups -dot- com``.
+  If it is off-hours, another ACIS Ops team member should call Peter
+  Ford, Jim Francis, and Bob Goeke.
```

Comment:
If we think this is the event going on, does Bob really need to be part of the initial software checks?

Reply:
I copied the notifications from another anomaly page and we could certainly discuss who gets notified when.  If we're diagnosing an unknown or not seen before anomaly, I think we would want the whole brain trust including Bob.  He may not be as expert as Peter and Jim, but is still more expert than the rest of us.